### ISSUE TITLE
Feedback on Project Proposal

### ISSUE THREAD

**Issue by rainerlempert** [UBC-MDS/Colourblind8#9]
Great job team! Very interesting concept, I'm a little surprised this doesn't exist yet! I also appreciate the included appendix. A few comments:
1. It would be good if you could more specifically define which changes to color schemes you intend to make with all three functions. Will you attempt to accommodate all of the points outlined in the appendix? If so fantastic! But if that is a bit too complicated/time consuming please let us know. This assignment shouldn't be too complicated to implement test cases.
2. It might also be a good idea to define exactly how many colors each function can support. Again, this could help limit the complexity of your functions.
3. It would be great to have a little more detail on how exactly these functions will modify the layout of the plots. 
4. Will these functions be used with a specific python plotting package, e.g. `matplotlib`? Or do you intend for this to work generally for all python plots?

**Comment by hfboyce** [comment]
Hi Rainer! 
1. We currently are still learning what changes are possible so this is something we are learning as we proceed further with the project. We mostly will be changing the colours of the objects in the plots, the axis text sizes and colours. 

2 . In terms of design, the colours will be chosen in milestone 2 when we have done a bit more research and have a better grasp of what is possible. 

3.  We are still in the design phase of this and will update you as we proceed. Right now we are assuming objects, font sizes, legends and more. Our test cases demonstrate a few of the expected changes we will be making.  

4.  This is intended for matplotlib specifically as stated in the proposal.
